### PR TITLE
Build and preview comments.

### PR DESCRIPTION
**Pull request by jmcarp**
* Build comments as markdown using Django templates
* Expose preview endpoint
* Pass document ID to templates

cc @vrajmohan

### CODE REVIEW

File: regulations/templates/regulations/comment.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{% spaceless %}
```
Comment (jmcarp) [inline regulations/templates/regulations/comment.md]: Also asking for opinions about this in #django, but django templates might not be the best choice for building markdown. This produces a bit of extra whitespace, which can be cleaned up using some inelegant hacks like deleting newlines or writing something like

```
{% endfor %}{% comment %}
{% endcomment %}{% endfor %}
```

@vrajmohan @cmc333333

Comment (cmc333333) [inline regulations/templates/regulations/comment.md]: I'd be in favor of adding jinja2 for this; I like it much more than django's template language.

Comment (jmcarp) [inline regulations/templates/regulations/comment.md]: :100:

File: regulations/static/regulations/js/source/views/comment-review-view.js [hunk 2]
```diff
@@ -49,7 +49,7 @@ var CommentReviewView = Backbone.View.extend({
 
   serialize: function() {
     return {
-      docNumber: this.docNumber,
+      doc_number: this.docNumber,
```
Comment (vrajmohan) [inline regulations/static/regulations/js/source/views/comment-review-view.js]: Can we please stick with the name documentId? That's how it is referred to in regulations.gov.

Comment (jmcarp) [inline regulations/static/regulations/js/source/views/comment-review-view.js]: This is probably a good time to resolve https://github.com/eregs/notice-and-comment/issues/23. The value here (so far) is the FR document number, which might or might not be the same as the regs.gov document ID. Do you know how the FR and regs.gov IDs are related?

Comment (vrajmohan) [inline regulations/static/regulations/js/source/views/comment-review-view.js]: I don't. I can look into eregs/notice-and-comment#23. However, for the MVP, as it is only a single document, we can get by with just "configuring" it.

Comment (cmc333333) [inline regulations/static/regulations/js/source/views/comment-review-view.js]: Regs.gov doesn't actually use `documentId` when submitting (they use `comment_on`), so let's do this:

1) call this field `fr_id` or something similar
2) ignore it and use a setting in `regulations.tasks.submit_comment`

I think the frontend will have the `comment_on` id some point soon, but there's no point delaying this.

**Comment by cmc333333** [comment]
Nice

**Pull request merged**